### DISCUSSION TITLE
Sponsors

### DISCUSSION THREAD

**Discussion by The-Compiler** [pytest-dev/sprint#5] [General]
Probably the easiest of all questions I'm asking here: Who should we list as sponsors of the sprint (say, in this repo, on-site in welcome slides, etc. etc.)?

Personal opinion, though I'm obviously wearing two different hats here:

- [Omicron](https://www.omicronenergy.com/en/) as they provide the location and food
- [Bruhin Software](https://bruhin.software/) (my own company) since I'm pouring a lot of time into organizing this thing, other than my contact at Omicron mostly as the only person.
- No other (monetary) sponsors, simply because we have the funds and I lack the time to solicit more sponsors as well. Companies might (hopefully!) pay for travel/accommodation for people, but they already get a lot back in comparison to that IMHO.

Thoughts?

**Comment by nicoddemus** [comment]
> Companies might (hopefully!) pay for travel/accommodation for people, but they already get a lot back in comparison to that IMHO.


But does not hurt to mention those companies as Sponsors? I mean, it might encourage other companies to do the same in the future.

**Reply by The-Compiler** [reply]
I'm mostly concerned with the additional time overhead communicating with n companies regarding logos, guidelines around it, and all the things companies like you to consider if displaying their logo. It'd be completely fine IMHO to just include the company names in the form of "Thanks to Acme Corporation and Umbrella Corporation for paying for their employees' travel/accommodation".

**Reply by nicoddemus** [reply]
I see, yeah I agree this would be definitely fine for now. :+1:

**Reply by flub** [reply]
yeah, all sounds reasonable to me
